src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -187,6 +187,8 @@ enum {
     UCT_IB_MLX5_MD_FLAG_CQE128_ZIP           = UCS_BIT(11),
     /* Device performance is optimized when RDMA_WRITE is not used */
     UCT_IB_MLX5_MD_FLAG_NO_RDMA_WR_OPTIMIZED = UCS_BIT(12),
+    /* Device supports indirect xgvmi MR */
```

Comment:
/* Device supports indirect xgvmi MR. This flag is removed if xgvmi access command fails */